### PR TITLE
[PXCT-81] Nano Matter - Technical check with 2.1.0 Silabs Core and User Manual Update

### PR DESCRIPTION
**Pull request by mcmchris**
## What This PR Changes
- Technical check with 2.1.0 Silabs Core and User Manual Update

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jcarolinares** [comment]
jumping to review this

File: content/hardware/03.nano/boards/nano-matter/tutorials/user-manual/content.md [hunk 1]
```diff
@@ -1169,24 +1171,151 @@ The `ble_initialize_gatt_db()` function is responsible for:
...
+
+After uploading the sketch to the Nano Matter, it is time to communicate with it via Bluetooth® Low Energy. For this, Silicon Labs has developed a **mobile app** that you can download from here:
+
+- [Android](https://play.google.com/store/apps/details?id=com.siliconlabs.bledemo)
+- [iOS](https://itunes.apple.com/us/app/silicon-labs-blue-gecko-wstk/id1030932759)
 
-![Blinky demo controlling the Nano Matter through Bluetooth® Low Energy](assets/BlinkyBLE.gif)
+Open the **Simplicity Connect** app on your smartphone, in the lower menu, navigate to **Demo** and select **Blinky**:
 
-***You can also manage the LED control and button status manually from the Scan tab in the lower menu.***
```
[the remark this answers is not in the document]
Comment (mcmchris) [reply content/hardware/03.nano/boards/nano-matter/tutorials/user-manual/content.md:1181]: Added and reformated

**Comment by mcmchris** [comment]
@jcarolinares thanks for your feedback, I applied your suggestions!

**Review by jcarolinares — APPROVED** [review]
Feedback applied, ready to merge @mcmchris

**Comment by mcmchris** [comment]
Merging!

**Pull request merged**